packages/polyfills/tsconfig.json

```diff
@@ -2,8 +2,8 @@
   "extends": "../tsconfig_base.json",
   "compilerOptions": {
     "outDir": "./dist",
-    "baseUrl": ".",
-    "rootDir": "."
+    "baseUrl": "./src",
```

Comment:
Why doesn't adding `src` to the polyfills prefix fix this without changing the tsconfig?

https://github.com/Shopify/quilt/blob/6a937f9576c6b4398ab1bee5ac6f41bbb4bfdd89/packages/polyfills/src/index.ts#L35

Reply:
I attempted that and re-ran tests in `email`. Tests still fail.

Reply:
Works for me.  Retest, please.

Reply:
Got it working, had to update it in sewing-kit's node_modules as well